src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -145,6 +145,17 @@ AppHost::~AppHost()
 
     _showHideWindowThrottler.reset();
 
+    // From a mail thread: Should a UnhandledException be raised by XAML, while
+    // we're tearing down, let's eat it, log it, and just die immediately. This
+    // will prevent unhandled XAML exceptions from crashing with a dump bucket
+    // when the window has already been closed.. We feel this is okay though,
+    // because WE'RE ALREADY IN TEARDOWN! The app is exiting. We don't want a
+    // XAML bug here to create a crash bucket.
+    _app.UnhandledException([](auto&& /*sender*/, const UnhandledExceptionEventArgs& args) {

```

Comment:
Couple things!

1. I'd rather not do this directly in the destructor, because destructors should be all about resource destruction.
2. `std::exit()` _may still trigger the destruction of static objects_; you could end up recursively destructing `AppHost`, for example, if it's in static storage.